src/themes/website/styles/forms.scss

```diff
@@ -281,4 +281,14 @@ $search-button-width: 30px;
 }
 .input-password::-ms-reveal {
     display: none;
-}
\ No newline at end of file
+}
+
+.remove-button {
```

Comment:
Since this style related to buttons, I'd suggest to move it to `buttons.scss` (but please read the previous comment first).